src/MF_Modules/MFSegments.cpp

```diff
@@ -4,16 +4,17 @@
 
 #include "MFSegments.h"
 
+#define _initialized  (_moduleCount != 0)
```

Comment:
interesting, i personally find it is not as obvious to read in the code later with this inline replacement ... is there a convention in terms of UPPERCASE or similar? 

I would rather replace it with _moduleCount != 0 in the  places where we are using it or have a function that we call which will be replaced inline by the compiler later.

Reply:
You're right, capitalized would be better. And given it's small, even direct substitution would be OK.
An inline would probably be overkill.